src/storage/v2/indices/text_index.cpp

```diff
@@ -251,10 +278,6 @@ void TextIndex::CreateIndex(std::string const &index_name, LabelId label, storag
 void TextIndex::RecoverIndex(const std::string &index_name, LabelId label,
```

Comment:
check with product:
- Why do we delete all durability files to recover from text index? Should be possible to load tantivy data.